### PR TITLE
Feature/update dependencies

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Updates @pxblue/angular-auth-template to use ^2.0.0 of the auth-workflow and updates example
- Toolbars must specify primary color
- Updates all other pxblue dependencies to latest.

### CODE REVIEW

**Comment by emclaug2** [comment]
I think we should delete v1.0.1 of @pxblue/angular-template-authentication since upgrading to ^2.0.0 of the auth-workflow will break the template.

File: src/app/auth-workflow/CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+### Changed
+
+-   Updated @pxblue/angular-auth-workflow dependency to version 2.0.0 (breaking-change)
+
 ## v1.0.1
 
 ### Changed
 
 -   Updated @pxblue/angular-themes dependency to version 6.0.1
 -   Updated @pxblue/angular-components dependency to version 4.2.0
```
Comment (joebochill) [inline src/app/auth-workflow/CHANGELOG.md:14]: Since this version does not work, should we unpublish it and remove it from the changelog?

**Review by joebochill — APPROVED** [review]

File: src/app/auth-workflow/CHANGELOG.md (hunk 1 above)
Comment (emclaug2) [reply src/app/auth-workflow/CHANGELOG.md:14]: If we can unpublish it from npm then yep I think we should remove this from the Changelog entirely.

Comment (joebochill) [reply src/app/auth-workflow/CHANGELOG.md:14]: It is now unpublished.

Comment (emclaug2) [reply src/app/auth-workflow/CHANGELOG.md:14]: Perfect, thanks @joebochill  I'll update the changelog to remove the 1.0.1

**Pull request merged**